tools/build/CreateRelease.php

```diff
@@ -31,7 +31,7 @@
 $consoleWrite = new ConsoleWriter();
 $lineSeparator = PHP_EOL;
 
-if (php_sapi_name() !== 'cli') {
+if (!Tools::isPHPCLI()) {
```

Comment:
It's not possibile because this script is run out of the PrestaShop scope. Tools is not accessible.

Reply:
Oh, I see.